pkg/vulnsrc/vulnerability/vulnerability.go

```diff
@@ -216,7 +218,10 @@ func getRejectedStatus(details map[types.SourceID]types.VulnerabilityDetail) boo
 		if !ok {
 			continue
 		}
-		if strings.Contains(d.Description, rejectVulnerability) {
+		if strings.Contains(d.Description, rejectVulnerability) || strings.Contains(d.Description, rejectedDoNotUse) {
```

Comment:
Ideally, we should take the status from `vulnStatus`.

```
jq .vulnStatus api/2024/CVE-2024-4109.json
"Rejected"
```

```suggestion
		// TODO(NVD): Take the vulnerability status from `vulnStatus`
		if strings.Contains(d.Description, rejectVulnerability) || strings.Contains(d.Description, rejectedDoNotUse) {
```

Reply:
hi @knqyf263 
I have updated PR to check rejected VulnStatus for NVD source 
Also further checking for other resources if we have reject keywords in description.
Because
I found some for amazon advisory. [here](https://github.com/aquasecurity/vuln-list/blob/main/amazon/2/ALAS2-2023-2088.json)

my fork trivy build with logs to print rejected vuln from other sources: [here](https://github.com/gshanbhag525/trivy-db/actions/runs/15709004652/job/44262431981)

can you take a quick look.